cmd/difference.go

```diff
@@ -133,6 +151,12 @@ func difference(sourceClnt, targetClnt Client, sourceURL, targetURL string, isRe
 
 			current := urlJoinPath(targetURL, srcSuffix)
 			expected := urlJoinPath(targetURL, tgtSuffix)
+
```

Comment:
We can move the `matchExcludeOptions` if block from here to above before line#134, so that `matchExcludeOptions` if block inside `if tgtEOF {` on line#134 can be eliminated. 
Line number #149 also needs to be moved up for the above change to work.

Reply:
Done.